global.json

```diff
@@ -3,8 +3,10 @@
     "dotnet": "2.1.401"
   },
   "msbuild-sdks": {
+    "Microsoft.DotNet.Arcade.Sdk": "1.0.0-beta.19061.6",
```

Comment:
You're adding extra entries to this file. You just need to delete the 2 additions and leave, 19064.4 entries.

Reply:
Oops, must have slipped in during a merge. I'll fix it.